debian/control

```diff
@@ -63,3 +66,12 @@ Description: JPEG XL Image Coding System - "JXL" (shared libraries)
  several features that help transition from the legacy JPEG format.
  .
  This package installs shared libraries.
+
+Package: libjxl-gimp
```

Comment:
I was just discussing this with @veluca93 . What do you think about `libjxl-gimp-plugin` for the binary package?

Reply:
(no strong opinions for me here)

Reply:
I'm fine with that... another option was gimp-libjxl, since I noticed most gimp plugins are named gimp-*  I'll change it to libjxl-gimp-plugin